C-Plus-Plus/math/Euler's_Totient_function.cpp

```diff
@@ -0,0 +1,26 @@
+#include <bits/stdc++.h>
```

Comment:
Add description of Euler's Totient Function.

Reply:
Sir, I have added Euler's Totient Function description just below function code.

Reply:
@atarax665 Just checking, would a ' in the programs name cause any issues?

Reply:
> @atarax665 Just checking, would a ' in the programs name cause any issues?

@VijayaGB98 No, it would be fine. Only spaces are encoded as %20 in URL.